cmd/client.go

```diff
@@ -173,6 +173,10 @@ type Client interface {
 
 	// Restore an object
 	Restore(ctx context.Context, versionID string, days int) *probe.Error
+
+	// OD operations
+	ODGet(ctx context.Context, part int) (io.ReadCloser, *probe.Error)
+	PutN(ctx context.Context, reader io.Reader, size int64, progress io.Reader, opts PutOptions) (n int64, err *probe.Error)
```

Comment:
if this is specific to OD rename each functions appropriately, you can write it as 

```
GetPart()
PutPart()
```